rpcs3/Emu/RSX/Common/ranged_map.hpp

```diff
@@ -177,63 +177,63 @@ namespace rsx
 			std::for_each(m_metadata.begin(), m_metadata.end(), [&](auto& meta) { meta.id = static_cast<u32>(&meta - m_metadata.data()); });
 		}
 
-		inline void emplace(const utils::address_range& range, T&& value)
+		void emplace(const utils::address_range& range, T&& value)
 		{
 			broadcast_insert(range);
 			m_data[block_for(range.start)].insert_or_assign(range.start, std::forward<T>(value));
 		}
 
-		inline iterator find(const u32 key)

```

Comment:
Seems like the methods `find` and `at` have been swapped unintendedly.

Reply:
I removed `at` and added `count` to make it compatible with at32 requirements